hack/jenkins/verify-jenkins-jobs.sh

```diff
@@ -38,6 +38,13 @@ kubernetes-pull-build-test-e2e-gce
 kubernetes-pull-test-unit-integration
 kubernetes-update-jenkins-jobs
 kubernetes-verify-jenkins-jobs
+kubernetes-upgrade-1.0-current-release-gce
+kubernetes-upgrade-gce-1.1-master
+kubernetes-upgrade-gce-stable-current-release
+kubernetes-upgrade-gke-1.0-current-release
+kubernetes-upgrade-gke-1.0-master
+kubernetes-upgrade-gke-1.1-master
+kubernetes-upgrade-gke-stable-current-release
```

Comment:
Yep. I'm not sure if it's worth keeping this script around after everything is YAML.

I'd still like to stop using multijobs, though.